LeftNavBarExample/LeftNavBarLibrary/build.gradle

```diff
@@ -3,7 +3,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.0.0'
+        classpath 'com.android.tools.build:gradle:2.2.1'
```

Comment:
why these changes to other modules? I see we have a root multi-module project, but what's the value of it? ideally all these demos should be separated project, right?
If anything we should consider to extract this common dependency (or at least the version) in a single place where it will be easier to maintain